src/shared/components/oncoprint/DeltaUtils.spec.ts

```diff
@@ -27,7 +27,10 @@ describe('Oncoprint DeltaUtils', () => {
         it('should return 2 for one empty input and one with two (both added/deleted)', () => {
             assert.equal(
                 numTracksWhoseDataChanged(
-                    [{ key: 'a', data: [] }, { key: 'b', data: [] }],
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki